src/utils.c

```diff
@@ -374,37 +374,39 @@ SEXP coerceUtf8IfNeeded(SEXP x) {
   return(ans);
 }
 
-// Convert a character vector into a single string, e.g. for printing error messages
+// Concatenate a character vector into a single string, e.g. for printing error messages
 // adapted from https://stackoverflow.com/a/58163237
 // make sure to *free* the returned string after use
-char *joinCharVec (SEXP x, const char *sep)
+// # nocov start
```

Comment:
why whole function is escaped from nocov? aren't we able emit error that uses this function in a reproducible way?